integration-tests/oauth2-boot2/pom.xml

```diff
@@ -38,7 +38,7 @@
         <dependency>
             <groupId>org.springframework.security.oauth.boot</groupId>
             <artifactId>spring-security-oauth2-autoconfigure</artifactId>
-            <version>${spring-boot.version}</version>
+            <version>2.0.1.RELEASE</version>
```

Comment:
Why do you use a variable on line 32, then hard-code it here? Also, should it be the latest (2.0.2)?

Reply:
This dep doesn't have the same release versions as the rest of spring boot, i should probably add a comment about that though (i'm also assuming it will get rolled into the bom at some point)